Bottom: ca0782162ab0201bfcc1d1bc4c2bfe129b9ea0f6
Top:    2ca74dcf2839db0641d982acca9ca6f777c3dda2
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-04-20 12:27:58 -0400

Refresh of mpc5200-ac97

---

diff --git a/sound/soc/fsl/Kconfig b/sound/soc/fsl/Kconfig
index 449464b..c532d8a 100644
--- a/sound/soc/fsl/Kconfig
+++ b/sound/soc/fsl/Kconfig
@@ -35,6 +35,7 @@ config SND_SOC_MPC5200_AC97
 	tristate "Freescale MPC5200 PSC in AC97 mode driver"
 	depends on SND_MPC52xx_SOC && PPC_BESTCOMM
 	select SND_SOC_OF_SIMPLE
+	select SND_MPC52xx_DMA
 	select PPC_BESTCOMM_GEN_BD
 	help
 	  Say Y here to support the MPC5200 PSCs in AC97 mode.
